### DISCUSSION TITLE
Can blueprint monitoring "check" settings please support retry count and timeout configuration?

### DISCUSSION THREAD

**Discussion by mixels** [gameserverapp/Platform#974] [Suggestions]
Sometimes modded game servers are slow to respond to query and RCON requests. Sometimes they don't respond at all. These kinds of issues tend to arise in heavily modded games in particular. We've seen it in Ark Evolved, Ark Ascended, and Conan Exiles.

It would be nice if we could use query or RCON monitoring with the option to control timeout and retry count in the GSA blueprint. Timeout would be the time GSA waits for a response when querying the game server and waiting for the response before deciding no response was received, and retry count would be the number of times GSA retries if previous checks have failed before deciding the server is down.

Container monitoring isn't a perfect solution either because the app running does not necessarily mean it's responding. Again, mods can cause this issue in some games. This is why I prefer to use query monitoring, but I can't right now for these games because GSA too often kills my servers when it thinks they're down (but they're up, and players are playing).

This feature would make query and RCON monitoring much more robust.

**Comment by mvaessen** [comment]
Hi,

GSA has this already setup for all game servers. So this is all automatically taken care of. I currently have no plans to let users mess with those numbers.
